.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Feature request
```

Comment:
Can I test it somehow? Or it must be deployed (to master I guess as it is the default branch)?